build.sbt

```diff
@@ -38,11 +38,11 @@ def commonSettings: Seq[Setting[_]] = Seq(
   )
 )
 
-def relaxNon212: Seq[Setting[_]] = Seq(
+def compilerVersionDependentScalacOptions: Seq[Setting[_]] = Seq(
   scalacOptions := {
     val old = scalacOptions.value
     scalaBinaryVersion.value match {
-      case "2.12" => old
+      case "2.12" => old ++ List("-opt-inline-from:<sources>", "-opt:l:inline", "-Yopt-inline-heuristics:at-inline-annotated")
```

Comment:
Due to https://github.com/sbt/zinc/issues/537, does this mean that incremental compilation of Zinc itself no longer works?